pkg/config/model.go

```diff
@@ -11,7 +11,6 @@ type APIConfig struct {
 	EnableAnimatedThumbnails bool   `mapstructure:"enable-animated-thumbnails" json:"enable-animated-thumbnails"`
 	MaxThumbnailSize         uint   `mapstructure:"max-thumbnail-size" json:"max-thumbnail-size"`
 
-	TwitchUsernameCacheDuration      time.Duration `mapstructure:"twitch-username-cache-duration" json:"twitch-username-cache-duration"`
```

Comment:
We will want to add a changelog entry noting that this is no longer used, or some runtime warning, before just removing it

Reply:
I feel like it makes more sense to just use this config entry since it already exists. I added an entry to the example config.yaml to document it.